oneview_redfish_toolkit/category_resource.py

```diff
@@ -18,11 +18,10 @@
 import threading
 
 
-# Globals vars:
-#   globals()['map_category_resources_ov']
-
 lock = threading.Lock()
 
+map_category_resources_ov = None
```

Comment:
We could init with dict here and remove the `init_map_category_resources()` and is not necessary use global before the reference of variable inside the methods. 
Buut no problem with this details, PR approved.